docs/navigation-configuration.md

```diff
@@ -1,242 +1,263 @@
+      ]
+    },
+    {
+      pathSegment: 'TopNav2',
+      label: 'Top Navigation Element Two',
+      viewUrl: 'https://2ndmicrofronted.com',
+      children: [
+...
+```
 
-If you set the **hideSideNav** property to `true`, the left navigation disappears when you click the affected node. It is set to `false` by default.
+## Basic navigation parameters
 
-If you want to group some navigation nodes into a separate parent node, you can use the **category** property. The grouped navigation nodes are rendered in a dropdown. The **category** property needs a **label** and, optionally, an **icon**.
+>**NOTE:** For a full list of available parameters, see the [parameter reference](navigation-parameters-reference.md) document.
 
-A sample navigation structure looks as follows:
+The [first steps](#first-steps) example provides some basic navigation parameters:
 
-````
-{
-  navigation: {
-    nodes: [
-      {
-        pathSegment: 'home',
-        label: 'Home',
-        viewUrl: 'https://my.microfrontend.com/',
-        children: [
-          {
-            link: '/home',
-            label: 'Go back home'
-          },
-          {
-            link: 'projects/pr2/settings',
-            label: 'Go to Project 2 Settings'
-          },
-          {
-            pathSegment: 'settings',
-            label: 'Settings',
-            viewUrl: 'https://my.microfrontend.com/general/settings.html'
-          },
-          {
-            pathSegment: 'projects',
-            label: 'Projects',
-            viewGroup: 'projectsGroup',
-            viewUrl: 'https://my.microfrontend.com/projects/list.html',
-            children: [
-              {
-                pathSegment: 'pr1',
-                label: 'Project one',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=pr1',
-                hideSideNav: true
-              },
-              {
-                pathSegment: 'pr2',
-                label: 'Project two',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=pr2'
-              }
-            ]
-          },
-          {
-            category: {label:'Misc', icon:'miscellaneous'},
-            pathSegment: 'miscellaneous',
-            label: 'Miscellaneous',
-            viewUrl: 'https://my.microfrontend.com/general/miscellaneous.html'
-          },
-          {
-            category:'Misc',
-            pathSegment: 'miscellaneous2',
-            label: 'Miscellaneous2',
-            viewUrl: 'https://my.microfrontend.com/general/miscellaneous2.html'
-          }
-        ]
-      }
-    ]
-  }
-}
-````
+### pathSegment
+This is used to build the path in the browser URL. The main application path is built from values in the navigation path, joined with the **/** character. For example, if the value of a node's **pathSegment** is `home`, the path for that node will be `yourwebsite.com/home`. You can override this setting by using one of the following instead of **pathSegment**: 
```

Comment:
```suggestion
This is used to build the path in the browser URL. The main application path is built from values in the navigation path, joined with the **/** character. For example, if the value of a node's **pathSegment** is `home`, the path for that node would be `yourwebsite.com/home`. You can override this setting by using one of the following instead of **pathSegment**: 
```